pkgs/development/python-modules/labgrid/default.nix

```
@@ -32,13 +33,9 @@ buildPythonPackage rec {
     sha256 = "sha256-yhlBqqCLOt6liw4iv8itG6E4QfIa7cW76QJqefUM5dw=";
   };
 
-  patches = [
-    # Pyserial within Nixpkgs already includes the necessary fix, remove the
-    # pyserial version check from labgrid.
-    ./0001-serialdriver-remove-pyserial-version-check.patch
-  ];
+  nativeBuildInputs = [ setuptools-scm pythonRelaxDepsHook ];
```

```suggestion
  nativeBuildInputs = [
    setuptools
    setuptools-scm
    wheel
    pythonRelaxDepsHook
  ];
```
Could you please add `setuptools` and `wheel` explicitly?
https://github.com/labgrid-project/labgrid/blob/447f4af921143fa3abcf6785f5564f3c24aee1f9/pyproject.toml#L1-L8